resources/views/packages/create.blade.php

```diff
@@ -0,0 +1 @@
+"Create Packages is here"
```

Comment:
-  Content should be wrapped  <div class="wrapper">

-  @extends('layouts.app')

- @section('content')